.circleci/config.yml

```diff
@@ -537,6 +542,56 @@ jobs:
           fi
           make check-links
 
+  owasp_image_scan:
```

Comment:
I would omit this unless you can actually confirm that it does anything useful with the scratch image that we use.

Reply:
> I would omit this unless you can actually confirm that it does anything useful with the scratch image that we use.

This does seem to catch some issues if I don't use the suppression.xml below. I need to investigate some more to see if the issues are actually valid or false positives.

Reply:
https://circleci.com/gh/signalfx/signalfx-agent/153822#tests/containers/ and https://153822-80042799-gh.circle-artifacts.com/0/home/circleci/project/test_output/dependency-check-report.html show some vulnerabilities found (the job currently only fails if critical vulns are found).